### PR TITLE
Select columns

### LINKED ISSUE #13: Add ability to select columns
We would like to have something that allows the user to select columns, it could be something similar to this selection experience that can be seen in many apps
Image

### PR DESCRIPTION
**Pull request by juanse1904**

### CODE REVIEW

File: src/renderer/pages/modelling-ui-poc/column-select/column-select.tsx [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+    const grid = new InfiniteGrid({
+      spacingX: 10,
+      spacingY: 10,
+      color: new Color(COLORS.GRAY_SOFT),
+      distance: 2000,
+    });
+
+    grid.position.set(0, 0, -0.0001); // little tweak to make sure the grid stays behind all objects
+
+    // scene.add(grid);
```
Comment (mapra99) [inline src/renderer/pages/modelling-ui-poc/column-select/column-select.tsx:100]: this PoC would still work if we uncomment this, right?

File: src/renderer/pages/modelling-ui-poc/column-select/column-select.tsx [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+    scene.add(modelGrid.object3D);
+    const columnGroup = ColumnsGroup({ columns });
+    scene.add(columnGroup);
+
+    window.addEventListener('click', onPointerMove, false);
+    function animate() {
+      requestAnimationFrame(animate);
+      controls.update();
+      // cssControls.update();
+      // cssRenderer.render(scene, camera);
```
Comment (mapra99) [inline src/renderer/pages/modelling-ui-poc/column-select/column-select.tsx:183]: this PoC would still work if we uncomment this, right?

**Review by mapra99 — APPROVED** [review]
go go go!

**Comment by mapra99** [comment]
closes #13

File: src/renderer/pages/modelling-ui-poc/column-select/column-select.tsx (hunk 2 above)
Comment (juanse1904) [reply src/renderer/pages/modelling-ui-poc/column-select/column-select.tsx:183]: I would say yes, but I'll check it

File: src/renderer/pages/modelling-ui-poc/column-select/column-select.tsx (hunk 1 above)
Comment (mapra99) [reply src/renderer/pages/modelling-ui-poc/column-select/column-select.tsx:100]: it works :)

File: src/renderer/pages/modelling-ui-poc/column-select/column-select.tsx (hunk 2 above)
Comment (mapra99) [reply src/renderer/pages/modelling-ui-poc/column-select/column-select.tsx:183]: it works :), just tested it

**Pull request merged**